src/MarkovChainMonteCarlo.jl

```diff
@@ -214,14 +221,20 @@ end
 
 
 function proposal(mcmc::MCMC)
-
     proposal_covariance = get_cov(mcmc.prior)
- 
+    prop_dist = MvNormal(zeros(length(mcmc.param)),
+                         (mcmc.step[1]^2) * proposal_covariance)
```

Comment:
I think the the `prop_dist` should be just 
```julia
prop_dist = MvNormal(zeros(length(mcmc.param)), proposal_covariance)
```
then for rwm we have
```julia
sample = mcmc.posterior[:,1 + mcmc.iter[1]] .+ (mcmc.step[1])*rand(prop_dist)
```
otherwise the pCN covariance (which also calls `rand(prop_dist)`) will be incorrectly scaled?

Reply:
Thanks for catching this! Fixed in d8183f8. Was very shortsighted to mindlessly copy the pre-existing code, since stepsize scaling is the **entire point** of implementing pCN!

I don't believe there's a way we could write a unit test to verify this in the future, right? The best we could do would be "run pCN with a small stepsize, and check it doesn't take absurdly long to converge."

Reply:
Yeah I don't think there is a way round this over analyzing the mixing times at different dimensions.